### ISSUE TITLE
zero scores

### ISSUE THREAD

**Issue by antgr** [namisan/mt-dnn#170]
I have used :
BERT_PATH="checkpoints/mt-dnn-rte_adamax_answer_opt1_gc0_ggc1_2020-04-26T0324/model_0.pt"
and the scores for dev set and test set are zero.

Seems that the error goes up:
04/26/2020 03:38:34 At epoch 0
04/26/2020 03:38:34 Task [ 0] updates[     1] train loss[0.56227] remaining[0:13:10]
04/26/2020 03:39:24 Task [ 5] updates[   500] train loss[0.60000] remaining[0:11:02]
04/26/2020 03:40:15 Task [ 3] updates[  1000] train loss[1.04799] remaining[0:10:12]
04/26/2020 03:41:06 Task [ 0] updates[  1500] train loss[1.98074] remaining[0:09:22]
04/26/2020 03:41:56 Task [ 2] updates[  2000] train loss[2.98191] remaining[0:08:31]
04/26/2020 03:42:47 Task [ 1] updates[  2500] train loss[3.95106] remaining[0:07:40]
04/26/2020 03:43:37 Task [ 0] updates[  3000] train loss[5.01362] remaining[0:06:49]
04/26/2020 03:44:28 Task [ 5] updates[  3500] train loss[6.07541] remaining[0:05:59]
04/26/2020 03:45:19 Task [ 2] updates[  4000] train loss[7.19914] remaining[0:05:08]
04/26/2020 03:46:09 Task [ 2] updates[  4500] train loss[8.43027] remaining[0:04:17]
04/26/2020 03:46:59 Task [ 4] updates[  5000] train loss[9.66741] remaining[0:03:27]
04/26/2020 03:47:50 Task [ 1] updates[  5500] train loss[10.88766] remaining[0:02:36]
04/26/2020 03:48:41 Task [ 1] updates[  6000] train loss[12.26786] remaining[0:01:45]
04/26/2020 03:49:31 Task [ 4] updates[  6500] train loss[nan] remaining[0:00:55]
04/26/2020 03:50:20 Task [ 3] updates[  7000] train loss[nan] remaining[0:00:04]

**Comment by namisan** [comment]
the training loss is quite odd, you may need to check your settings.
